gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -214,6 +214,17 @@ def get_predetermined_qc(version: str = CURRENT_VERSION, test: bool = False):
     },
 )
 
+# v3 and v4 combined sample metadata Table for relatedness and population inference
+qc_meta = VersionedTableResource(
```

Comment:
maybe a name like "joint_qc_meta" would make it more clear that this doesn't just consist of v4 samples?